### ISSUE TITLE
Serialization of (unavailable)Future can lead to thread stuck

### ISSUE THREAD

**Issue by artpar** [unloggedio/unlogged-sdk#52]
### Describe the bug

its getting stuck here inside file : DetailedEventStreamAggregatedLogger

```
else if (value instanceof Future) {
                        Future<?> futureValue = (Future<?>) value;
                        bytes = objectMapper.get().writeValueAsBytes(futureValue.get());
```

### Reproduction steps

1. hazelcast with future
2. 
3. 
...

### Expected behavior

application behaviour should remain unaffected

**Comment by tushar886** [comment]
The issue is happening at this piece of code in Unlogged SDK 

```
else if (value instanceof Future) {
                        Future<?> futureValue = (Future<?>) value;
                        bytes = objectMapper.get().writeValueAsBytes(futureValue.get());
```
its getting stuck here inside file : DetailedEventStreamAggregatedLogger

**Comment by artpar** [comment]
should be resolved with 0.5.1
